site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc

```diff
@@ -5,7 +5,7 @@ The list of Link objects.
 endif::[]
 
 [field]#{base_field_name}.displayName# [type]#[String]#::
-A human readable name for this link to help you identify this link. This value will generally be an email address, or username. This value is recorded when the link is made, and updated during the next login for this identity provider. This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider.
+A human readable name for this link for identification and convenience. This value will generally be an email address, or username. This value can be set by the API caller. So for instance, this value could be `Initial piedpiper link from user migration`. This value will be updated during the next login for the linked user based on the identify provider (`richard@piedpiper.com`, for instance). This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider.
```

Comment:
We should preserve consistency. So if you are going to use `jared@piedpiper.com` in the example response, you should use that here as well, instead of `richard`.